docs/faq.rst

```diff
@@ -146,9 +146,9 @@ really desperate (e.g. if you have no completed backup of that file and you'ld
 rather get a partial file extracted than nothing). You do **not** want to give
 that option under any normal circumstances.
 
-Note that checkpoints inside files are created only since version 1.1, 
+Note that checkpoints inside files are created only since version 1.1 
```

Comment:
did you intentionally remove the comma? why?

Reply:
By mistake; sorry. Not used to editing files using the GitHub UI.